src/collection-browser.ts

```diff
@@ -291,10 +291,11 @@ export class CollectionBrowser
   /** Whether to replace the default sort options with a slot for customization (default: false) */
   @property({ type: Boolean }) enableSortOptionsSlot = false;
 
-  /** Whether to display a smart results carousel above the full results */
+  /** Whether to expose a slot for a smart results carousel above the full results */
   @property({ type: Boolean, reflect: true }) showSmartResults = false;
 
-  @property({ type: String }) resultsHeader?: string;
+  /** Header content to display above the results view */
+  @property({ type: String }) resultsHeader?: TemplateResult | string;
```

Comment:
Q: Since this is being put in a slot, would it make sense to just use the slot from the caller instead of passing in the `string` or `TemplateResult`, ie:
```
<collection-browser>
  <div slot="results-heading">
    <h1>Boop</h1>
  </div>
</collection-browser>
```
If not, the `type` should probably be an `Object` since `TemplateResult` isn't a string.

Reply:
Yeah, I suppose it's a bit odd. The intent behind keeping both the property and the slot was to avoid this being a breaking change. As it is now, any existing `resultsHeader` being passed in will still continue to work as long as nothing else gets slotted in there. But maybe we should just bite the bullet and remove the property entirely.

Regardless, the `TemplateResult` type can be removed here either way -- just relic from a first draft that I backed out of!